<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>561</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>American Sign Language</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>British Sign Language</string>
     </attribute>
     <widget class="QLabel" name="nombrecont">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>110</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Precision</string>
      </property>
     </widget>
     <widget class="QLabel" name="matchresultlabel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;La lettre est :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="signname">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>140</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Tolerance LVL:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="tolerance_lvl">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>250</y>
        <width>60</width>
        <height>27</height>
       </rect>
      </property>
      <property name="value">
       <number>9</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>324</x>
        <y>240</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>ok</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="Delais">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>121</width>
        <height>29</height>
       </rect>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="value">
       <number>40</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Expressions</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Add new data</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>180</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="imageboxname">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Enter the name of the new image</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sign Language</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Webcam</string>
    </property>
   </widget>
   <widget class="QPushButton" name="template_buttom">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>View Hand</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>574</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuIceman">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="exit"/>
   </widget>
   <addaction name="menuIceman"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
